README.md

```diff
@@ -58,6 +59,9 @@ return ECSConfig::configure()
     ->withRules([
         ArraySyntaxFixer::class,
     ])
+    // Warnings for included PHPCS rules are disabled by default,
+    // but they can be enabled manually.
+    ->withWarnings()
```

Comment:
Hey, thanks for many contributions :partying_face: 

Let's take it one by one to ship them faster :)
I like this feature - please separate it to own PR :+1:

Reply:
Sure thing!

Just so we're on the same page, do you just want me to pull out the `with[out]Warnings` methods/parameters with warnings disabled by default, then enabled when using `withSnifferStandards`?

Reply:
Thanks 👍 

Lets start with warnings as a standalone feature. Without any dependency on `withSnifferStandards`.

Reply:
Sorry for the delay, been focused on "work" work! I'll be able to get back to this soon, and I'll create four separate PRs for (in this order):

1. The additional test script.
1. The cache invalidation fix.
1. Duplicate error reports for PHPCS rules.
1. Missing logic to skip self-disabled PHPCS rules during parsing.
1. Add support for PHPCS warnings.

I'll make sure each is appropriately tested, of course. I'll then rebase this branch and force merge to my fork. Hopefully that'll cut down on the effort required to review these submissions.😄 

Let me know if you're opposed to the plan above or have any suggestions. Thanks for your time!

Reply:
4/5 of the above have been prepared and are ready for your review! The last changes to allow PHPCS warnings has dependencies on the others though, so I'll need to wait for those PRs to be merged before coming back to finish it.

I appreciate your time, Tomas! Sorry again for taking so long. 😭